client/css/components/forms/_registrationForm.css

```diff
@@ -1,4 +1,2 @@
 #registrationForm {
```

Comment:
no reason. this file can also be removed if none custom style for login and register form is included.